aries_cloudagent/multitenant/admin/routes.py

```diff
@@ -205,12 +224,20 @@ async def wallet_create(request: web.BaseRequest):
     key_management_mode = body.get("key_management_mode") or WalletRecord.MODE_MANAGED
     wallet_key = body.get("wallet_key")
 
+    # wallet_dispatch_type
+    # default: dispatch only to webhook_urls defined
+    # base: dispatch only to webhook_targets
+    # both: dipatch to both
     settings = {
         "wallet.type": body.get("wallet_type") or "in_memory",
         "wallet.name": body.get("wallet_name"),
         "wallet.key": wallet_key,
+        "wallet.webhook_urls": body.get("wallet_webhoook_urls") or [],
+        "wallet.dispatch_type": body.get("wallet_dispatch_type") or "default",
```

Comment:
Maybe we can use `admin.webhook_urls` here? This way we can keep the send webhook method "dumb".

Based on the dispatch_type we will extend the base context differently (`get_wallet_profile` method in `MultitenantManager`):
- both: `admin.webhook_urls` = combination of base webhook urls and the webhook urls stored in the wallet_record
- base: do not store webhook_urls in wallet record. When extending the webhook urls of the base settings will be used
- default: do store in webhook urls in wallet record and overwrite settings when extending context

This way we don't need to store any new settings, and don't have to do any custom logic to determine if we're in base of subwallet context.

Reply:
@TimoGlastra I appreciate the feedback, I have made some changes to align with your suggested approach. 
- I also had to make changes to the send_webhook function [change to original logic is minimal], as the target URLs are now available in profile.settings.get("admin.webhook_urls") v/s webhook_targets dictionary which has only has the base URLs. 
- Attributes such as max_attempt and topic_filter can also be supported, if you think this approach is good then I can go about handling these.